flowclient/flowclient/query_specs.py

```diff
@@ -840,3 +840,23 @@ def random_sample_spec(
     )
     sampled_query["sampling"] = sampling
     return sampled_query
+
+
+def per_subscriber_aggregate_spec(*, subscriber_query: Dict, agg_method: str):
+    """
+    Query that performs per-subscriber aggregation of a table. Returns a column
+        'subscriber' containing unique subscribers and a column 'value' containing the
+        aggregration.
+
+        Parameters
+        ----------
+        subscriber_query: SubscriberFeature
+            A query with a `subscriber` column
+        agg_method: {"count", "sum", "avg", "max", "min", "median", "stddev", "variance"} default "avg"
+            The method of aggregation to perform
```

Comment:
Re-adding my comment here following the file move, for visibility:
>The descriptions of the parameters here need updating to reflect the parameters of the flowclient function, rather than of the flowmachine class.